src/bin/internallauncher

```diff
@@ -112,13 +112,23 @@ class JobSubmitter(object):
 #   Call the new PPNArgument function to get the correct MPI argument for 
 #   the "process per node" flag.
 #
+#   Eric Brugger, Fri Oct  9 15:41:37 PDT 2020
+#   Modify the routine to use our mpirun if we are running an installed
+#   version and it exists.
+#
 ###############################################################################
 
 class JobSubmitter_mpirun(JobSubmitter):
     def __init__(self, launcher):
         super(JobSubmitter_mpirun,self).__init__(launcher)
 
     def Executable(self):
+        # Use our mpirun if we are running an installed version and it
+        # exists.
+        if self.launcher.generalArgs.dv == 0:
```

Comment:
Am curious but what is the `dv` member?

Reply:
That indicates that is running a development version (dv) instead of an installed version. When you run VisIt after building it, "-dv" gets magically added to the command line for all the components when you run "bin/visit".